packages/matter.js/src/cluster/ClusterHelper.ts

```diff
@@ -4,69 +4,85 @@
+    [Clusters.LowPowerCluster.id]: Clusters.LowPowerCluster,
+    [Clusters.MediaInputCluster.id]: Clusters.MediaInput.Complete,
+    [Clusters.MediaPlaybackCluster.id]: Clusters.MediaPlayback.Complete,
+    [Clusters.ModeSelectCluster.id]: Clusters.ModeSelect.Complete,
+    [Clusters.NetworkCommissioningCluster.id]: Clusters.NetworkCommissioning.Complete,
+    [Clusters.OccupancySensingCluster.id]: Clusters.OccupancySensingCluster,
+    [Clusters.OnOffCluster.id]: Clusters.OnOff.Complete,
+    [Clusters.OnOffSwitchConfigurationCluster.id]: Clusters.OnOffSwitchConfigurationCluster,
+    [Clusters.OperationalCredentialsCluster.id]: Clusters.OperationalCredentialsCluster,
+    [Clusters.OtaSoftwareUpdateProviderCluster.id]: Clusters.OtaSoftwareUpdateProviderCluster,
+    [Clusters.OtaSoftwareUpdateRequestorCluster.id]: Clusters.OtaSoftwareUpdateRequestorCluster,
+    [Clusters.PowerSourceCluster.id]: Clusters.PowerSource.Complete,
+    [Clusters.PowerSourceConfigurationCluster.id]: Clusters.PowerSourceConfigurationCluster,
+    [Clusters.PressureMeasurementCluster.id]: Clusters.PressureMeasurement.Complete,
+    [Clusters.ProxyConfigurationCluster.id]: Clusters.ProxyConfigurationCluster,
+    [Clusters.ProxyDiscoveryCluster.id]: Clusters.ProxyDiscoveryCluster,
+    [Clusters.PulseWidthModulationCluster.id]: Clusters.PulseWidthModulation.Complete,
+    [Clusters.PumpConfigurationAndControlCluster.id]: Clusters.PumpConfigurationAndControl.Complete,
+    [Clusters.RelativeHumidityMeasurementCluster.id]: Clusters.RelativeHumidityMeasurementCluster,
+    [Clusters.ScenesCluster.id]: Clusters.ScenesCluster,
+    [Clusters.SoftwareDiagnosticsCluster.id]: Clusters.SoftwareDiagnostics.Complete,
+    [Clusters.SoilMoistureMeasurementCluster.id]: Clusters.SoilMoistureMeasurementCluster,
+    [Clusters.SwitchCluster.id]: Clusters.Switch.Complete,
+    [Clusters.TargetNavigatorCluster.id]: Clusters.TargetNavigatorCluster,
+    [Clusters.TemperatureMeasurementCluster.id]: Clusters.TemperatureMeasurementCluster,
+    [Clusters.ThermostatCluster.id]: Clusters.Thermostat.Complete,
+    [Clusters.ThermostatUserInterfaceConfigurationCluster.id]: Clusters.ThermostatUserInterfaceConfigurationCluster,
+    [Clusters.ThreadNetworkDiagnosticsCluster.id]: Clusters.ThreadNetworkDiagnostics.Complete,
+    [Clusters.TimeFormatLocalizationCluster.id]: Clusters.TimeFormatLocalization.Complete,
+    [Clusters.TimeSyncCluster.id]: Clusters.TimeSync.Complete,
+    [Clusters.UnitLocalizationCluster.id]: Clusters.UnitLocalization.Complete,
+    [Clusters.UserLabelCluster.id]: Clusters.UserLabelCluster,
+    [Clusters.ValidProxiesCluster.id]: Clusters.ValidProxiesCluster,
+    [Clusters.WakeOnLanCluster.id]: Clusters.WakeOnLanCluster,
+    [Clusters.WiFiNetworkDiagnosticsCluster.id]: Clusters.WiFiNetworkDiagnostics.Complete,
+    [Clusters.WindowCoveringCluster.id]: Clusters.WindowCovering.Complete,
```

Comment:
Hrm not as easy to populate programmatically as I was thinking.  I'll add something to make that easier but we can do after merge.

Reply:
Yes I also wanted to ask. But ok for now.